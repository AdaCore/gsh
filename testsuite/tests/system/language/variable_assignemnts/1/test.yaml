title: Check variable assignment
driver: shell_script
