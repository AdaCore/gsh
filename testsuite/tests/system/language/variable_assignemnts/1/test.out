TOTO=PASSED
TATA=PASSED
TOTO should be empty: 
